project-canvas/src/components/backlog-view/Issue/DetailView.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+  Accordion,
+  Avatar,
+  Badge,
+  Box,
+  Button,
+  Grid,
+  Group,
+  Menu,
+  Paper,
+  Stack,
+  Text,
+  ThemeIcon,
+  Title,
+} from "@mantine/core"
+import { IconBinaryTree2 } from "@tabler/icons"
+import { Issue } from "project-extender"
+import { IssueIcon } from "./IssueIcon"
+
+export function DetailView({
+  issueKey,
+  summary,
+  status,
+  storyPointsEstimate,
+  epic,
+  labels,
+  assignee,
+  creator,
+  description,
+  sprintId,
+  subtasks,
+  created,
+  updated,
+  comment,
+  type,
+}: Issue) {
+  return (
+    <Paper p={10}>
+      <Group mb={40} sx={{ gap: "8px" }} color="dimmed" fs="italic">
+        <IssueIcon type="Epic" />{" "}
+        {epic !== undefined ? (
+          <Text>{epic}</Text>
+        ) : (
+          <Text tt="uppercase">Add Epic</Text>
+        )}{" "}
+        /
+        <IssueIcon type={type} /> <Text>{issueKey}</Text>
+      </Group>
+      <Grid>
+        <Box w="60%" sx={{ overflow: "auto", maxHeight: "550px" }}>
+          <Title order={2} mb={30}>
+            {summary}
+          </Title>
+          <Text color="dimmed">Description</Text>
+          {description !== "" ? (
+            <Group mb={30}>
+              <Text>{description}</Text>
+            </Group>
+          ) : (
+            <Text color="dimmed" mb={30} fs="italic">
+              Add Description
+            </Text>
+          )}
+
+          <Text color="dimmed" mb={10}>
+            Child Issues
+          </Text>
+          {subtasks.length !== 0 ? (
+            <Paper mb={30} withBorder>
+              <Stack spacing="xs">
+                {subtasks.map((subtask) => (
+                  <Paper
+                    withBorder
+                    key={subtask.id}
+                    p={5}
+                    sx={(theme) => ({ display: "flex", gap: theme.spacing.md })}
+                  >
+                    <ThemeIcon size="sm" mt={2}>
+                      <IconBinaryTree2 />
+                    </ThemeIcon>
+                    <Text size="md" w="90%">
+                      <Text size="sm" color="blue" span mr="md">
+                        {subtask.key}
+                      </Text>
+                      {subtask.fields.summary}{" "}
+                    </Text>
+                  </Paper>
+                ))}
+              </Stack>
+            </Paper>
+          ) : (
+            <Text color="dimmed" mb={30} fs="italic">
+              Add child Issue
+            </Text>
+          )}
```

Comment:
check the margins, they are different between (Description-Add Description) and (Child Issues-Add child Issue)